File: GerenciaMusic360/ClientApp/app/shared/models/configuration-project-task-contract.ts
import { IContractType } from "./contractType";

export interface IConfigurationProjectTaskContract {
    id: number;
    projectTypeId: number;
    contractTypeId: number;
    taskDocumentDetailId: number;
    required: number;
    configurationId: number;
    contractType: IContractType;
}

File: GerenciaMusic360.Entities/ConfigurationProjectTaskContract.cs
namespace GerenciaMusic360.Entities.Models
{
    public class ConfigurationProjectTaskContract
    {
        public int Id { get; set; }
        public int ProjectTypeId { get; set; }
        public short ContractTypeId { get; set; }
        public int? TaskDocumentDetailId { get; set; }
        public bool Required { get; set; }
        public short ConfigurationId { get; set; }

        public virtual ContractType ContractType { get; set; }
    }
}
